charts/osm-arc/templates/osm-label.yml

```diff
@@ -1,6 +1,6 @@
-# 
-# The job in this YAML file tries to add 'openservicemesh.io/ignore' label to 'kube-system', 'azure-arc', and 'arc-osm-system' namespaces.
-# The job is executed in helm pre-install phase of OSM release lifecycle. 
+#
+# The job in this file adds 'openservicemesh.io/ignore' label to 'kube-system', 'azure-arc', and 'arc-osm-system' namespaces.
```

Comment:
We should also add that the job adds the "admission.policy.azure.com/ignore" label to the arc-osm-system namespace.